src/componants/ViewSingleArticle.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from 'react';
+import { getSingleArticle } from '../../utils/apiRequests';
+import { useParams } from 'react-router-dom';
+
+const ViewSingleArticle = () => 
+{
+    const [articleBody, setArticleBody] = useState([]);
+    const { article_ID } = useParams();
```

Comment:
You could add a state for loading so a message could be displayed to the user while the article is loading. Using conditional rendering.